send-data/ingest.mdx

```diff
@@ -284,3 +284,13 @@ All events stored in Axiom must have a `_time` timestamp field. If the data you
 </Note>
 
 If you include the `_time` field in the ingested data, ensure the `_time` field contains timestamps in a valid time format. Axiom accepts many date strings and timestamps without knowing the format in advance, including Unix Epoch, RFC3339, or ISO 8601.
+
+## Best practices for ingesting data
+
+When ingesting data into Axiom, follow these best practices to optimize performance and reliability:
+
+- Batch events: Instead of sending individual events, group multiple events into a single request. This reduces the number of API calls and improves overall throughput.
```

Comment:
Does this refer to doing it on the app itself or to configuring your log forwarder to do it? If the later :+1:, if the former, its complicated as there are several considerations there :)

Reply:
@pecigonzalo I’m not following "on the app itself". Can you help?

Reply:
Sure! 

When we say batch the logs before sending here, are we saying to use some system or pattern to batch them? eg. `app <- otel-collector -> axiom` in which case otel-collector does the batching

or we are saying, in your application buffer the logs and then write them to axiom as a batch?